scripts/_lines_of_code.py

```diff
@@ -65,11 +77,21 @@ def __init__(self, owned_repository: Repository, user):
         self.total_lines_of_addition = 0
         self.total_lines_of_deletion = 0
         self.name = owned_repository.name
-        for contribution in owned_repository.get_stats_contributors():
-            if contribution.author.id == user.id:
-                for week in contribution.weeks:
-                    self.total_lines_of_addition += week.a
-                    self.total_lines_of_deletion += week.d
+
+        if (owned_repository.get_commits().totalCount >= 100):
+            execute_bind('<inform>', 'This repository has a lot of commits! This can take some time...')
+
+        for commit in owned_repository.get_commits():
+        if commit.author is not None:
```

Comment:
Resolve indentation error.
```suggestion
            if commit.author is not None:
```